app/src/lib/stores/app-store.ts

```diff
@@ -4898,6 +4903,25 @@ export class AppStore extends TypedBaseStore<IAppState> {
     return Promise.resolve()
   }
 
+  public _resizeSidebarWidth(resizeDirection: ResizeDirection): Promise<void> {
+    const { value, min, max } = this.sidebarWidth
+    const changedWidth =
+      resizeDirection === ResizeDirection.Decrease ? value - 1 : value + 1
```

Comment:
Could be more aggressive than 1? This hardly is noticeable if you use the menu item... but allows fine control if using keyboard shortcuts? Maybe menu item should have a higher factor?